package.json

```diff
@@ -29,7 +29,8 @@
     "rename": "sh scripts/changeName.sh",
     "genRadix": "sh scripts/generateRadixConfig.sh",
     "prepare": "husky install",
-    "postinstall": "sh scripts/postInstall.sh"
+    "postinstall": "sh scripts/postInstall.sh",
+    "fresh-install": "rm -rf package-lock.json && rm -rf node_modules && npm install"
```

Comment:
We're using [yarn](https://yarnpkg.com/) as the package manager in `Flyt`. Since this script generates a new lockfile it should have a more descriptive name:
```JSON
"genLockfile": "rm yarn.lock && rm -rf node_modules && yarn"
```

Reply:
👍